components/core/JsInterop/modules/dom/manipulationHelper.ts

```diff
@@ -242,6 +242,31 @@ export class manipulationHelper {
     styleHelper.removeCls(document.body, "ant-scrolling-effect");
   }
 
+  static focusOnChildFormIfExists(parentId) {
+      let BreakException = {};
+      let parent = document.getElementById(parentId);
+      if (parent) {
+          let forms = parent.getElementsByTagName('form');
+          if (forms.length > 0) {
+              let form = forms[0];
+              let inputs = form.querySelectorAll('input, textarea, select');
+              try {
+                  inputs.forEach(function (input:HTMLElement) {
+                      var style = window.getComputedStyle(input);
+                      if (style.display !== 'none' && style.visibility !== 'hidden') {
+                          input.focus();
+                          throw BreakException;
```

Comment:
I don't think throwing an exception is a good way to exit the loop. You can use `inputs.find(input=> ... )` to find the input